readme.md

```diff
@@ -315,7 +315,20 @@ In addition to `redux` --> `mobx-state-tree`, we've also transitioned to using `
 
 In Bowser, TypeScript is fully set up, so if you know TS, all you need to do is start coding!
 
-### Resources
+## GraphQL comparing to RESTful
+
+### Design/Architecture
+RESTful API uses HTTP verbs (Post, Get, Patch, etc...) in order to manipulate data. 
+Because of this, designing a RESTful system requires creating endpoints for each request such as: POST/user, GET/user, DELETE/user etc... 
+
+GraphQL uses the HTTPS Post to send either Queries and Mutations to interact with a database. 
```

Comment:
@JonnoKiwi needs a transition like "On the other hand, , GraphQL [...]" or "In contrast, , GraphQL [...]" or "To make it easier, GraphQL [...]"